google/cloud/spanner_v1/database.py

```diff
@@ -216,6 +221,31 @@ def ddl_statements(self):
         """
         return self._ddl_statements
 
+    @property
+    def logger(self):
+        """Logger used by the database.
+
+        The default logger will log commit stats at the log level INFO using
+        `sys.stderr`.
+
+        :rtype: :class:`logging.Logger` or `None`
+        :returns: the logger
+        """
+        if self._logger is None:
+            self._logger = logging.getLogger(self.name)
+            self._logger.setLevel(logging.INFO)
+
+            ch = logging.StreamHandler()
+            ch.setLevel(logging.INFO)
+
+            formatter = logging.Formatter(
+                "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
```

Comment:
I don't know all the context, but are you sure all these fields would be useful to all users? (Is it necessary to put the levelname? It seems like all logs would have level `INFO`)

I'd suggest using a less opionated default, and letting the user add formatting if they need it